azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -0,0 +1,336 @@
+        final Class<?> hadoopSecurityManagerClass =
+                HadoopProxy.class.getClassLoader().loadClass(hadoopSecurityClassName);
+
+        logger.info("Loading hadoop security manager " + hadoopSecurityManagerClass.getName());
+        hadoopSecurityManager = (HadoopSecurityManager)
+                Utils.callConstructor(hadoopSecurityManagerClass, keyStoreLoadProps);
+      } catch (final Exception e) {
+        logger.error("Could not instantiate Hadoop Security Manager ", e);
+        throw new RuntimeException("Failed to get hadoop security manager!"
+                + e.getCause(), e);
+      }
+
+      final KeyStore keyStore = hadoopSecurityManager.getKeyStore(keyStoreLoadProps);
+      // Delete the cert file from disk
+      final File certFile = new File(Constants.ConfigurationKeys.CSR_KEYSTORE_LOCATION);
+      if (certFile.delete()) {
+        logger.info("Successfully deleted the cert file");
+      } else {
+        logger.error("Failed to delete the cert file");
+        throw new IOException("Failed to delete the cert file");
+      }
+      if (keyStore == null) {
+        logger.error("Failed to Prefetch KeyStore");
+        throw new IOException("Failed to Prefetch KeyStore");
+      }
+    }
+  }
+
+  public static void main(final String[] args) throws IOException, ExecutorManagerException {
+    // Redirect all std out and err messages into log4j
+    StdOutErrRedirect.redirectOutAndErrToLog();
+    // Get all the arguments
+    final String execIdStr = args[0];
+    final String projectZipName = args[1];
+    // Process Execution ID.
+    int execId = 0;
+    try {
+      execId = Integer.parseInt(execIdStr);
+    } catch (NumberFormatException ne) {
+      System.out.printf("Execution ID %s is invalid %n", args[0]);
+    }
+    System.out.printf("Execution ID = %d%n", execId);
+
+    // Setup work directories
+    final Path currentWorkingDir = Paths.get("").toAbsolutePath();
+    final Path projectDirPath = Paths.get(currentWorkingDir.toString(), PROJECT_DIR);
+    final Path jobtypePluginPath = Paths.get(currentWorkingDir.toString(), JOBTYPE_DIR);
+
+    // Move files to respective dirs
+    // Create project dir
+    // TODO: Need to revisit
+    System.out.println("Creating project dir");
+    Files.createDirectory(projectDirPath);
+    Path projectZipPath = Paths.get(projectDirPath.toString(), projectZipName);
+    System.out.println("moving projectDir:" + projectZipName + ": to " +
+        projectZipPath);
+    Files.move(Paths.get(projectZipName), projectZipPath);
+    // TODO : revisit this logic,
+    // Unzip the project zip
+    FlowContainer.unzipFile(projectZipPath.toString(), projectDirPath);
+
+    // TODO : Deepak - throw away after directory setup is final.
+    Path pwd = Paths.get("").toAbsolutePath();
+    System.out.println("Deepak : Dumping dir structure for = " + pwd.toString());
+    java.nio.file.Files.walk(pwd).filter(java.nio.file.Files::isRegularFile)
```

Comment:
Package java.nio.file.Files is already added. Can you please remove it from here?